src/main.js

```diff
@@ -1,4 +1,28 @@
+// Sentry
+import Vue from "vue";
+import * as Sentry from "@sentry/browser";
+import { Vue as VueIntegration } from "@sentry/integrations";
+import { Integrations } from "@sentry/tracing";
+// Vue
 import { createApp } from "vue";
 import App from "./App.vue";
 
+Sentry.init({
+  dsn: "https://eb054063b03849a0a393ecd06625f4a1@o135088.ingest.sentry.io/5473791",
```

Comment:
I know this is not a secret and will show up in JS so there is not technical problem with it being in the repo, but config stuff like this often feels better as ENV to me.

https://cli.vuejs.org/guide/mode-and-env.html#using-env-variables-in-client-side-code

A switch around this code based on the dsn being set in ENV would be ideal I think to make sure we don't accidentally load it for any builds we don't want it in.

Reply:
Oh, good point - there's a few things in this that I think would be better handled in config, yeah. I'll make this change here and create a ticket for the others (i.e. the TImdex URL)